source/getting-started/tutorials/sensors/index.md

```diff
@@ -16,25 +16,24 @@ This tutorial expands on the code created in the [Voltage Source Inverter](/gett
 
 ## Introduction 
 
-In motor control applications, it is important to have consistent timing of both sensor data acquisition and the running of control tasks. In default configuration, the AMDC allows for some jitter between sensor measurement time and control loop timing. This tutorial shows how the AMDC can be configured to elimiate this jitter.
+In motor control applications, it is important for the timing of both sensor data acquisition and control task execution to remain consistent. In its default configuration, the AMDC may allow some jitter (timing incosistency) between sensor measurement time and control loop timing. This tutorial shows how the AMDC's "Timing Manager" peripheral can be configured to eliminate this jitter.
 
-This tutorial provides the source code for:
-* experimenting the effects of the timing manager using the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) app
+This tutorial provides code that allows users to experiment with the configuration of the AMDC Timing Manager. The effects of this experimentation will be observed through the lense of the previous [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial.
 
 ## Scheduling and Synchronizing:
 
-The AMDC synchronizes running tasks and sensor collection to the PWM carrier wave. Every X PWM periods (where X is set by the function timing_manager_set_ratio()), the AMDC will collect data from sensors. In Legacy mode, the AMDC will run control tasks concurrently with sensor collection. In Post-sensor mode the AMDC will not run control tasks until the sensor collection is complete. 
+The AMDC synchronizes sensor data collection and task execution to the PWM carrier wave. Every X PWM periods (where X is set by the function `timing_manager_set_ratio()`), the AMDC will collect data from sensors. By default, the Timing Manager is set to Legacy Mode, in which the AMDC will run control tasks concurrently with sensor data collection. Alternatively, when configured in Post-Sensor Mode, the Timing Manager guarantees that in each control cycle, the AMDC will wait to execute control tasks until the sensor data collection is complete.s
 
 There are multiple factors that affect when and how fast control tasks run.
- - User set TASK_UPDATES_PER_SEC
+ - User set `TASK_UPDATES_PER_SEC`
  - PWM Frequency
  - User Event Ratio
- - Sensor collection time (post-sensor mode)
+ - Sensor collection time (Post-Sensor Mode)
  - Control task time (how long it takes for the control task to run)
 
- ![](images/timing.png)
+ ![](/firmware/arch/images/timing.png)
 
-Read [this docs page](/firmware/arch/timing-manager.md) for detailed information on the timing manager
+Read [this docs page](/firmware/arch/timing-manager.md) for detailed information on the Timing Manager
 
 Consider: Control tasks only have the opportunity to run once every X PWM periods, where X is the User Event Ratio.\
```

Comment:
I think this should probably say something like:

"Control tasks can run _at most_ once every control period"

since they can run less often if users so desire.